plugin/src/software/aws/toolkits/eclipse/amazonq/util/ProxyUtil.java

```diff
@@ -30,6 +37,22 @@ protected static String getHttpsProxyUrl(final String envVarValue, final String
         if (!StringUtils.isEmpty(prefValue)) {
             httpsProxy = prefValue;
         }
+        try {
+            if (!StringUtils.isEmpty(httpsProxy)) {
+                URI.create(httpsProxy);
+            }
+        } catch (IllegalArgumentException e) {
+            if (!hasSeenInvalidProxyNotification) {
+                hasSeenInvalidProxyNotification = true;
```

Comment:
If a user sees this the first time, closes the display. Goes back and changes the proxy url but still does fix the format correctly, will they not see this popup then?

Reply:
It's a good point. We don't currently evaluate or attempt to use the proxy URL until the IDE is restarted. This is an overall area for improvement - ideally we can hot swap to the new setting without the need for a restart. The current logic works within our existing constraints. Input validation would also help address this. I'll create a backlog item.